dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/servlet/DisabledServlet.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat.apps.servlet;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@SuppressWarnings("serial")
+@WebServlet("/testDisabledHttpTrace")
+public class DisabledServlet extends FATServlet {
+
+    public static final String APP_NAME = "TelemetryServletTestApp";
+    public static final String SIMPLE_SERVLET = "simple";
+    public static final String INVALID_TRACE_ID = "00000000000000000000000000000000";
+
+    @Inject
+    private HttpServletRequest request;
+
+    @Test
+    public void testSimpleServlet() throws Exception {
+        String scheme = request.getScheme();
+        String serverName = request.getServerName();
+        int serverPort = request.getServerPort();
+        URL url = new URL(scheme + "://" + serverName + ":" + serverPort + "/" + APP_NAME + "/" + SIMPLE_SERVLET);
+
+        String traceId = httpGet(url); // The servlet outputs the traceId
+
+        // The simple servlet will return the default invalid traceId
+        assertEquals(traceId, INVALID_TRACE_ID);
+
+    }
```

Comment:
This test is valid and I think we should keep it, but it doesn't really test what you're changing here.

I think the only way to actually find out if the change has broken something would be to enable trace and look for the "instrumenter is initialized" or "instrumenter is set to null" messages.

We could add that check to this test, but I think it's sufficient to check it yourself manually before merging this PR.

Reply:
Run with tracing enabled which did return "instrumenter is set to null"